### ISSUE TITLE
Pull request ?

### ISSUE THREAD

**Issue by hargathor** [clarus-proxy/JSqlParser#2]
Do you think that it will be usefull to merge these version of [JSqlParser](https://github.com/JSQLParser/JSqlParser) with the official version ? At least do a pull request ?

Otherwise on each build on the CLARUS proxy we will need to build this fork on the side since we cannot push it in maven central.

**Comment by tchevallier** [comment]
I think it could be useful indeed, but before going into this process, I should ask Frederic if he's interested to do it on his own name (as he is the only contributor for this evolution). Otherwise we could consider doing it ourselves, indeed. I'll let you know.

**Comment by dieriver** [comment]
I agree with Thierry, the final decision about issuing a pull request or not should go through Frédéric. Before submitting the pull request, please check whether the contributor should be Frédéric (as himself) or the company.

**Comment by hargathor** [comment]
Yes of course I totally agree that's why I'm opening this issue. In any case the author of the commit will be Frederic since it's written in the git tree. 
The real question is: does he have the time to do it ?

**Comment by tchevallier** [comment]
I'm asking him today. We'll see.

**Comment by tchevallier** [comment]
Just discussed with Frederic : he does not have time to do it, but he thinks it could be a good idea. However, the problem is that JsqlParser has probably changed much since our modifications for CLARUS ... So before issueing a pull request, we should start from the latest version and apply the changes : this may not be a trivial matter (i.e. work to estimate)

**Comment by hargathor** [comment]
I can check this out today or tomorrow
